csrc/dynamic_transform.cpp

```diff
@@ -36,19 +37,24 @@ DynamicTransformInitialInfo DynamicTransformInitialInfo::clone(
       cloned_info.dynamic_resizes_.push_back(ir_cloner.clone(op));
     }
   }
-  cloned_info.expr_eval_ = expr_eval_.clone(ir_cloner);
+  cloned_info.root_dynamic_vals_.reserve(root_dynamic_vals_.size());
+  for (const auto v : root_dynamic_vals_) {
+    if (v) {
+      cloned_info.root_dynamic_vals_.insert(ir_cloner.clone(v));
+    }
+  }
```

Comment:
Roto dynamic vals are not used after cloning currently, but I realized this was not being cloned.